packages/fela-dom/src/dom/rehydrate.js

```diff
@@ -18,7 +18,7 @@ export default function rehydrate(renderer: DOMRenderer): void {
   render(renderer)
 
   const idQuery =
-    renderer.rendererId.length > 0
+    typeof renderer.rendererId !== 'undefined' && renderer.rendererId.length > 0
```

Comment:
Thanks for the PR

Why do we need `typeof` here? it's not a variable that can cause a reference error. Isn't it enough to just do:
```suggestion
    renderer.rendererId && renderer.rendererId.length > 0
```

Reply:
True, old habit since when I saw Id I thought also it could be an int. I've made the change.